src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -45,7 +45,7 @@ export class UserFormComponent extends CdForm implements OnInit {
 
   userFormMode = UserFormMode;
   mode: UserFormMode;
-  allRoles: Array<UserFormRoleModel>;
+  allRoles: Array<any>;
```

Comment:
We must use  proper types always. Lets not put "any"

Reply:
suggested changes are done